src/primitives/coordinate.rs

```diff
@@ -92,15 +92,15 @@ impl Into<(f64, f64)> for Coordinate {
     }
 }
 
-impl Into<[f64; 2]> for Coordinate {
+impl Into<[f64; 2]> for Coordinate2D {
     fn into(self) -> [f64; 2] {
         [self.x, self.y]
     }
 }
 
-impl<'c> Into<&'c [f64]> for &'c Coordinate {
+impl<'c> Into<&'c [f64]> for &'c Coordinate2D {
```

Comment:
do we really need this? one could also do let x: [f64; 2] = (*x).into(). There will be no performance hit.

Reply:
It is not about performance, but there are APIs requiring slices of `f64`s.